the-experts-fe/src/components/loginForm/loginForm.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+
+export default function LoginForm() {
+  const [user, setUser] = useState({
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: ""
+  });
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    console.log(user);
```

Comment:
Please remove unnecessary console.log 🙏 I'm going to start charging 1₪ for everyone I find 😂

Reply:
haha i forgot